action.yml

```diff
@@ -72,6 +72,9 @@ inputs:
   prerelease-prefix:
     description: 'A prefix to apply to the prerelease version number'
     required: false
+  base-tag:
```

Comment:
yes base tag acts like the `from` tag for auto-version-bump. I think `base-tag` makes more sense for auto bump because internally we use the latest tag if `base-tag` input is not provided and conventional-changelog package also uses tags for the bump. what do you think?

Reply:
Yes, let's use it. I'll use the same one in my task as well.